src/main/java/frc/trigon/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -10,13 +10,18 @@
 import frc.trigon.robot.RobotContainer;
 import frc.trigon.robot.constants.RobotConstants;
 import frc.trigon.robot.poseestimation.poseestimator.PoseEstimatorConstants;
+import frc.trigon.robot.subsystems.swerve.swervemodule.SwerveModule;
 import org.trigon.hardware.RobotHardwareStats;
 import org.trigon.hardware.phoenix6.pigeon2.Pigeon2Gyro;
 import org.trigon.hardware.phoenix6.pigeon2.Pigeon2Signal;
 
 import java.util.function.DoubleSupplier;
 
 public class SwerveConstants {
+    public static final double
+            MAXIMUM_SPEED_METERS_PER_SECOND = RobotHardwareStats.isSimulation() ? 4.9 : 4.04502,
+            MAXIMUM_ROTATIONAL_SPEED_RADIANS_PER_SECOND = RobotHardwareStats.isSimulation() ? 12.03 : 12.03;
```

Comment:
should this be here?

Reply:
Why shouldn't it? It's mostly used in the Swerve class

Reply:
It's misc...